data-in-pipeline/service-manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "service.name": "data-in-pipeline",
+  "service.namespace": "data-fetching",
+  "team": "application",
+  "inputs": [
+    {
+      "type": "service",
+      "name": "rds"
+    },
+    {
+      "type": "service",
+      "name": "families-api"
+    },
+    {
+      "type": "s3",
+      "name": "cpr-production-document-cache"
+    }
+  ],
+  "outputs": [],
```

Comment:
minor nitpick: intention is to have outputs as all systems/etc this writes to so we can model forward dependencies as wlel as input dependencies like above. IIRC we write to RDS so that should be duplicated here? And do we write to the s3 bucket as well or just read?